### DISCUSSION TITLE
Question about searchlight significance tests

### DISCUSSION THREAD

**Discussion by Aceticia** [rsagroup/rsatoolbox#365] [Q&A] [answered]
Hi all. I have a number of subjects' beta maps and a weighted model which I plan to run searchlight on. People in my lab haven't done anything like this, and I'd really appreciate if yall can provide some feedbacks on these proposed methods:

1. I can try to do first level searchlight analysis for each subjects independently, then use standard non parametric inference such as [this provided by nilearn](https://nilearn.github.io/stable/modules/generated/nilearn.glm.second_level.non_parametric_inference.html#nilearn.glm.second_level.non_parametric_inference) to derive p value for the model similarity at each voxel.
2. Method 1 doesn't consider the variances produced by noise. Instead of running inference on the raw similarity map, we can find the explained variances of the model for each subject, and adjust that by the subject's noise ceilings obtained from CV. Assuming we use Pearson's r, we can achieve this with something like: (model similarity^2 - lower noise ceiling ^2) / (upper noise ceiling ^2 - lower noise ceiling ^2).
3. Method 2 does consider noise, but only considers within-subject noise ceiling. We can first obtain a noise ceiling map from all subjects, and then do the same adjustments like in 2.

If context helps, this is related to [this issue that I recently opened](https://github.com/rsagroup/rsatoolbox/issues/357).

**Comment by JasperVanDenBosch** [comment, marked as answer]
Alternatively, if you get the beta maps in standardised space, then for every centre you could feed all subjects' rdms into the inference function. This would give you an estimate of the models' performance and weights on the group level.
